### ISSUE TITLE
brew install fails

### ISSUE THREAD

**Issue by mattcomisso** [dral3x/homebrew-dral3x#1]
Thanks for making it!

- Probably related to the published version on brew (0.0.2) -

```
==> Installing stringslint from dral3x/dral3x
==> Cloning https://github.com/dral3x/StringsLint.git
Cloning into '/Users/m.comisso/Library/Caches/Homebrew/stringslint--git'...
Note: checking out '1cd188f070a088fd779958dc70a3181ca58d955e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

==> Checking out tag 0.0.2
HEAD is now at 1cd188f release 0.0.2
==> make prefix_install PREFIX=/usr/local/Cellar/stringslint/0.0.2 TEMPORARY_FOLDER=/private/tmp/stringslint-20190402-52733-zmvb0m/StringsLint.dst
Last 15 lines from /Users/m.comisso/Library/Logs/Homebrew/stringslint/01.make:
      _$s4Yams4NodeO8SequenceVSMAASMy03SubC0QzSny5IndexQzGciMTW.resume.0 in Node.Sequence.swift.o
      ...
  "_swift_updateClassMetadata2", referenced from:
      _$s20StringsLintFramework4FileCMr in File.swift.o
  "_swift_willThrow", referenced from:
      _$s10Commandant14CommandWrapperVyxGADSbs5Error_pIgggdzo_A2DSbsAE_pIegnndzo_sAERzlTRTA in Command.swift.o
      _$s10Commandant14CommandWrapperVyxGSis5Error_pIggdzo_ADSisAE_pIegnrzo_sAERzlTRTA in HelpCommand.swift.o
      _$s10Commandant10OrderedSetVyxGSTAAST32withContiguousStorageIfAvailableyqd__Sgqd__SRy7ElementQzGKXEKlFTW in OrderedSet.swift.o
      _$s6ResultAAO13dematerializexyKF in Result.swift.o
      _$s20StringsLintFramework14ComposedParserVAA011LocalizableE0A2aDP5parse4fileSayAA15LocalizedStringVGAA4FileC_tKFTW in ComposedParser.swift.o
      _$s20StringsLintFramework23ObjcParserConfigurationV5applyyyypKF in ObjcParserConfiguration.swift.o
      _$s20StringsLintFramework24SwiftParserConfigurationV5applyyyypKF in SwiftParserConfiguration.swift.o
      ...
ld: symbol(s) not found for architecture x86_64
make: *** [build_with_disable_sandbox] Error 1
```

**Comment by dral3x** [comment]
Hey Matteo,

thanks for pointing me to this issue. I'll give a look at it asap. 💪

**Comment by dral3x** [comment]
I've tested the installation with version 0.0.3 and Xcode 10.1 and it works correctly.
@mattcomisso Are you using the latest Xcode 10.2? If so that might be the source of the problem.

As soon as I update Xcode, I'll fix the issue

**Comment by dral3x** [comment]
@mattcomisso This bug has been fixed in version 0.0.4.
Please give it another try and let me know if that's correct also for you.

Thanks! And sorry for taking soooo long 🙏

**Comment by mattcomisso** [comment]
I can confirm it's now working :)
